pages/glossary/browsing-context-related-nodes.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Set of browsing context related nodes
+key: browsing-context-related-nodes
+---
+
+Two HTML [nodes](https://dom.spec.whatwg.org/#concept-node) are _browsing context related_ if:
```

Comment:
I think this can be simplified by saying that they need to have the same [top-level browser context](https://www.w3.org/TR/html50/browsers.html#top-level-browsing-context). I also wouldn't worry about whether or not something is activated. Nodes don't show up in the accessibility tree if they're not, and we're always looking for that condition.

Reply:
I think we need to worry about full activation.
Each document has an accessibility tree, even if it's not fully activated. So, nodes can share the top level browsing context, and be in the accessibility tree of an inactive document.